tests/phpunit/test-option.php

```
@@ -62,12 +62,251 @@ function test_option() {
 			),
 		);
 		foreach ( $test_array as $key => $value ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)